src/components/dropdownMenu/DropdownMenuSubContent.tsx

```diff
@@ -0,0 +1,38 @@
+import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
+import { useDevice } from "../App";
+import Stack from "../Stack";
+import { Island } from "../Island";
+import clsx from "clsx";
+
+const DropdownMenuSubContent = ({
+  children,
+  className,
+}: {
+  children?: React.ReactNode;
+  className?: string;
+}) => {
+  const device = useDevice();
+
+  const classNames = clsx(`dropdown-menu ${className}`, {
+    "dropdown-menu--mobile": device.isMobile,
```

Comment:
```suggestion
    "dropdown-menu-mobile": device.isMobile,
```
since `mobile` isn't a modifier